src/enforcer.ts

```
@@ -357,6 +357,32 @@ export class Enforcer extends ManagementEnforcer {
 
     return res.filter((n) => !inherits.some((m) => n === m));
   }
+
+  /**
+   * getUsersForRoleInDomain gets the users from name and domain.
+   *
+   * @param name the user.
+   * @param domain the domain.
+   * @return the users that for role in domain.
+   */
+  public async getUsersForRoleInDomain(name: string, domain: string): Promise<string[]> {
+    const rm = this.rmMap.get('g');
+    if (rm) return rm.getUsers(name, domain);
+    throw new Error("RoleManager didn't exist.");
```

Why throw an exception instead of returning something like "null"?